pallets/proposals/src/impls.rs

```diff
@@ -207,7 +210,7 @@ impl<T: Config> Pallet<T> {
                 .unwrap_or(&default_max_cap);
 
             ensure!(
-                max_cap == default_max_cap || max_cap >= new_contribution_value.clone(),
+                max_cap == default_max_cap || max_cap >= old_contribution.value.clone(),
```

Comment:
Looks like this should be new_contribution?
Any reason why its changed?

Reply:
Also no need to clone() see if you can borrow both with &. Cloning is a deep copy and is expensive.

Reply:
correct, it should be your total contribution i.e new contribution

Reply:
it's a mistake, let me fix it.